### ISSUE TITLE
🐛 weird ANSI output on a remote VPS setup 

### ISSUE THREAD

**Issue by ow0x** [dandavison/delta#1020]
- [ ] Please include the raw text output from git, so that we can reproduce the problem.
      (You can use `git --no-pager` to produce the raw text output.)
- [x] A screenshot of Delta's output is often helpful also.

Hi,

first off thank you for making this incredibly useful tool. Today I installed latest `git-delta` v0.12.1 through `cargo install` command on my Ubuntu 20.04 remote VPS which is based on aarch64 (ARM) architecture. I did the gitconfig configuration as laid out in https://dandavison.github.io/delta/configuration.html

However whenever I use `git diff` or `git log` and other similar commands which display diffs, the terminal would show weird ANSI output as visible in below screenshot instead of proper colored output. I am not sure why it is happening but I would be really grateful if someone can point me in right direction for a solution. Thank you!

![image](https://user-images.githubusercontent.com/24418520/159120117-be29d8e5-3489-4e4b-a3fc-12065fa88b6a.png)

my `~/.gitconfig`:
```
[pull]
    rebase = false

[core]
    pager = delta

[interactive]
    diffFilter = delta --color-only --features=interactive

[delta]
    features = decorations

[delta "interactive"]
    keep-plus-minus-markers = false

[delta "decorations"]
    commit-decoration-style = blue ol
    commit-style = raw
    file-style = omit
    hunk-header-decoration-style = blue box
    hunk-header-file-style = red
    hunk-header-line-number-style = "#067a00"
    hunk-header-style = file line-number syntax
```

the system info of my remote VPS is:
```ansi
ubuntu@owo64
------------
OS: Ubuntu 20.04.4 LTS aarch64
Host: KVM Virtual Machine virt-4.2
Kernel: 5.15.0-1001-oracle
Uptime: 2 days, 19 hours, 4 mins
Packages: 1507 (dpkg)
Shell: zsh 5.8.1
Terminal: /dev/pts/0
CPU: (1)
GPU: 00:01.0 Red Hat, Inc. Virtio GPU
Memory: 1125MiB / 5916MiB

```

The terminal I am using to connect/interact with my said remote VPS through SSH is alacritty v0.10.1 (2844606d)

Please let me know if I need to provide any additional info in case I missed it in this post, I am happy to provide it to debug and solve this issue. I also did search in current open+closed issues before opening this bug report to avoid any duplicates but I wasn't able to find similar issues. Sorry for any inconvenience!

**Comment by dandavison** [comment]
Hi @ow0x, I'm glad you appreciate delta. Let's get this sorted out!

I think the clue to what's going on is at the bottom of your screenshot where it says `MOST`. `most` is an alternative pager to `less`. Delta automatically sends its output to a pager, but delta expects that pager to be `less`. So I suspect you have an environment variable, or an alias, that is somehow selecting `most` to be the pager on your ubuntu machine.

Here are some things I'd check:

```shell
which less
which most
env | grep -i pager
env | grep -i less
```

Also I think you'll see delta work correctly if you explicitly pipe its output to less: `git log -p | delta | less`

If it's not obvious what's going on then please do post the output of the commands above here and we can both debug it.

**Comment by ow0x** [comment]
Hi, thank you very much for the prompt response and support! 😄 

I just ran the commands you suggested and they did indeed showed `most` was aliased for `less` in my `~/.zshrc`. I assume it came bundled with default setup of my VPS since I never installed `most`.
I removed it and now delta is returning output in rich colours for my remote VPS setup.

![image](https://user-images.githubusercontent.com/24418520/159127414-a298351f-29fc-4b3a-a8b2-726070c4f080.png)


Thank you for taking a moment of your valuable time to help me. This indeed solved my issue much faster than I possibly could on my own and this really means alot to me. I'll close this issue since it is now resolved. Have a nice day! 🙏🏼
